### PR TITLE
Klee sqlite3 merge fixes

### PR DESCRIPTION
**Pull request by rsprabery**
@kren1

Thanks for the sqlite3 changes. Due to some changes in the main KLEE branch, your [PR](https://github.com/google/fuzzbench/pull/281) will fail to merge.

Since I don't have write access to your fork, I pulled in your changes and fixed the merge conflicts here. 

We'll get @laurentsimon to re-review, but since [PR-281](https://github.com/google/fuzzbench/pull/281) is already approved, I think we can merge this in quickly and close that PR out.

### CODE REVIEW

**Comment by rsprabery** [comment]
@googlebot I consent.

**Comment by rsprabery** [comment]
@kren1 - would you mind commenting with `@googlebot I consent.` - Looks like the bot doesn't like the fact that someone else modified your original commit.

Thanks!

File: fuzzers/klee/fuzzer.py [hunk 1]
```diff
@@ -21,327 +21,348 @@
...
 def is_benchmark(name):
-    """Check if the benchmark contains the string |name|"""
-    benchmark = os.getenv("BENCHMARK", None)
-    return benchmark is not None and name in benchmark
+  """Check if the benchmark contains the string |name|"""
+  benchmark = os.getenv('BENCHMARK', None)
+  return benchmark is not None and name in benchmark
+
+
+def is_benchmark(name):
```
Comment (kren1) [inline fuzzers/klee/fuzzer.py:35]: This seems duplicated now

**Comment by kren1** [comment]
@googlebot I consent.

File: fuzzers/klee/fuzzer.py (hunk 1 above)
Comment (rsprabery) [reply fuzzers/klee/fuzzer.py:35]: Thanks for the catch!

**Pull request merged**